name_that_hash/prettifier.py

```diff
@@ -81,30 +81,30 @@ def pretty_print_one(self, objs: List, multi_print: bool):
             out += "\n[bold underline #2ec4b6]Least Likely[/bold underline #2ec4b6]\n"
 
             for i in rest:
-                out += self.turn_named_tuple_pretty_print(i) + " "
+                out += self.turn_named_tuple_pretty_print(i) + "\n"
 
         console.print(out)
         return out
 
     def turn_named_tuple_pretty_print(self, nt: NamedTuple):
         # This colours red
-        out = f"[bold #e71d36]{nt['name']}[/bold #e71d36], "

```

Comment:
There's a bug here where if it doesn't contain hashcat, john, or a description it's a hanging "-". I fixed this with commas by just manually checking the input hahaha

Reply:
![image](https://user-images.githubusercontent.com/10378052/106600535-7899c880-6552-11eb-96c0-2399e9ccd3f4.png)

Ah, with commas it is natural to not have any extra info but with dashes it is not. I was wondering how come I hadn't seen this bug 😅

Reply:
Understood! I took a little extra liberty than I had expected because I was on a roll, but I see your point about having to scroll up to see the Most Likely. As for the 'no match', what about not printing the entire line if all of the variables are null? This will also reduce the length.

Have you made the above changes already, or do you want me to make them in a follow-on to this PR?

Reply:
Hrm.. not sure that did what I wanted it to. Can you confirm on your end that the conflict resolved and changes are made? I was looking to merge my changes into main, not the other way around.

If push comes to shove, I'll just delete and re-submit with the proper changes.

Reply:
> As for the 'no match', what about not printing the entire line if all of the variables are null? This will also reduce the length.

For sure! If you can figure out how to do that 🥺 

> Hrm.. not sure that did what I wanted it to. Can you confirm on your end that the conflict resolved and changes are made?
Conflict resolved, but you're still using new lines 😄